### PR TITLE
added sound repeater

### PR DESCRIPTION
**Pull request by AgreS1r**

### CODE REVIEW

File: src/main/java/ru/sevsu/rc_manager/sound/processor/SoundRepeater.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    private void repeatSound(byte[] soundBytes, AudioFormat audioFormat) {
+        // Создаем новый AudioInputStream из массива байтов
+        ByteArrayInputStream bais = new ByteArrayInputStream(soundBytes);
+        AudioInputStream repeatedAudioInputStream = new AudioInputStream(bais, audioFormat, soundBytes.length / audioFormat.getFrameSize());
+
+        // Отправляем повторенный звук другим обработчикам
+        // Например, отправляем его в TelegramSendHandler
+        TelegramSendHandler telegramSendHandler = new TelegramSendHandler();
```
Comment (lantushenkoao) [inline src/main/java/ru/sevsu/rc_manager/sound/processor/SoundRepeater.java]: I don't think we need to send to telegram anything here

File: src/main/java/ru/sevsu/rc_manager/sound/processor/SoundRepeater.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        // Читаем аудиоданные из AudioInputStream
+        ByteArrayOutputStream bos = new ByteArrayOutputStream();
+        byte[] buffer = new byte[frameSize];
+        int bytesRead;
+        try {
+            while ((bytesRead = audioInputStream.read(buffer, 0, frameSize)) != -1) {
+                bos.write(buffer, 0, bytesRead);
+            }
+        } catch (IOException e) {
+            // Обработка исключения
```
Comment (lantushenkoao) [inline src/main/java/ru/sevsu/rc_manager/sound/processor/SoundRepeater.java]: add logging here

File: src/main/java/ru/sevsu/rc_manager/sound/processor/SoundRepeater.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
+package ru.sevsu.rc_manager.sound.handler;
+
+import javax.sound.sampled.AudioFormat;
+import javax.sound.sampled.AudioInputStream;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+// Обработчик получает массив байтов и метаданные файла wav
+public class RepeaterHandler implements Handler {
```
Comment (DarthTapik) [inline src/main/java/ru/sevsu/rc_manager/sound/processor/SoundRepeater.java]: Public class name and file name must be the same

File: src/main/java/ru/sevsu/rc_manager/sound/processor/SoundRepeater.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    @Override
+    public void handle(AudioInputStream audioInputStream) {
+        // Получаем метаданные из AudioInputStream
+        AudioFormat audioFormat = audioInputStream.getFormat();
+        int frameSize = audioFormat.getFrameSize();
+        int frameRate = (int) audioFormat.getFrameRate();
```
Comment (DarthTapik) [inline src/main/java/ru/sevsu/rc_manager/sound/processor/SoundRepeater.java]: Variable frameRate is not used

**Pull request merged**